content/docs/command-reference/exp/run.md

```diff
@@ -23,9 +23,13 @@ Provides a way to execute and track <abbr>experiments</abbr> in your
 <abbr>project</abbr> without polluting it with unnecessary commits, branches,
 directories, etc.
 
-> `dvc exp run` has the same behavior as `dvc repro` when it comes to `targets`
-> and stage execution (restores the dependency graph, etc.). See the command
-> [options](#options) for more on the differences.
+`dvc exp run` has the same general behavior as `dvc repro` when it comes to
+`targets` and stage execution (restores the dependency graph, etc.). One
+important difference is that changed dependencies are committed to the <abbr>DVC
+cache</abbr> when the experiment begins execution, to guarantee the
+reproducibility of any recorded experiment.
```

Comment:
One question for this PR's reviewer(s): #2418 was about performance implications. Should this be stressed further here or in the guide?

> For now I may just link to `gc` and check whether that ref. mentions `exp gc`(as vice versa).

Reply:
I think just noting that it is committed is fine

Reply:
See my comment in https://github.com/iterative/dvc.org/issues/2418#issuecomment-1036405900. `dvc repro` also commits changed dependencies to the DVC cache, so I don't see a difference here.

I'm having flashbacks to our original discussion about this 😅 . I think this initially came up because `dvc exp run` didn't used to commit changed dependencies, but we changed it to behave more like `dvc repro`.

Reply:
> this initially came up because dvc exp run didn't used to commit changed dependencies, but we changed it to behave more like dvc repro 

Right I just went all the way back to https://github.com/iterative/dvc/issues/5593, https://github.com/iterative/dvc/pull/5859, and #2418 and it seems that's what the change was. So this paragraph about being different to `repro` is incorrect... ⌛

Reply:
OK I updated the changes...

> I think just noting that it is committed is fine

Yeah that's what I did now. I didn't specifically mention it in the context of queueing, just left generic language that applies to running or queueing ("when preparing the experiment").